docs/src/content/en/docs/agents/sanitizing-input.mdx

```diff
@@ -238,46 +234,52 @@ class MessageLengthLimiter implements Processor {
       if (error instanceof TripWire) {
         throw error; // Re-throw tripwire errors
```

Comment:
i noticed you've developed a convention that mostly avoids code comments (I like it) not sure you need the comments here